shared_sql_utils/src/dsn.rs

```diff
@@ -55,7 +54,7 @@ pub struct Dsn {
     pub user: String,
     pub server: String,
     pub driver_name: String,
-    pub log_level: String,
+    pub log_level: Option<String>,
```

Comment:
Hmm.. do we need any notion of loglevel in dsn now?

Reply:
That's a good point -- I wasn't sure since the two are interlinked in some places, but seems like setting through the odbc uri works fine even if its removed from dsn. Made this change 👍